spec/jobs/replication/integration_spec.rb

```diff
@@ -65,7 +58,9 @@
     )
 
     # creating or updating a MoabRecord should trigger its parent PreservedObject to replicate any missing versions to any target endpoints
-    create(:moab_record, preserved_object: preserved_object, version: version, moab_storage_root: moab_storage_root)
+    perform_enqueued_jobs do
+      create(:moab_record, preserved_object: preserved_object, version: version, moab_storage_root: moab_storage_root)
+    end
```

Comment:
Make sure the enqueued jobs are run.

Reply:
or maybe we didn't realize we had to do this, and that's what the `:inline` stuff was making up for? 🤷

Reply:
Probably? I'm guessing this code pre-dated the `:test` adapter becoming the default, which works hand in hand with the `*enqueued_jobs` helpers.